Details from the one test failure (recording it here before the CI logs disappear):
```
/Users/ec2-user/jenkins/workspace/swift-PR-macos/branch-main/swift/validation-test/Reflection/reflect_Enum_value.swift:372:16: error: CHECK-NEXT: expected string not found in input
// CHECK-NEXT: Value: .child(_)
               ^
<stdin>:185:45: note: scanning from here
(enum reflect_Enum_value.OneIndirectPayload)
                                            ^
<stdin>:186:1: note: possible intended match here
Value: .child(.leafF)
^
```

--- a/validation-test/Reflection/reflect_Enum_value.swift
+++ b/validation-test/Reflection/reflect_Enum_value.swift
@@ -369,7 +369,7 @@ reflect(enumValue: OneIndirectPayload.child(.leafF))
 // CHECK: Reflecting an enum value.
 // CHECK-NEXT: Type reference:
 // CHECK-NEXT: (enum reflect_Enum_value.OneIndirectPayload)
-// CHECK-NEXT: Value: .child(_)
+// CHECK-NEXT: Value: .child(.leafF)
 
 reflect(enumValue: OneIndirectPayload.leafF)
 

